api/libs/blm-calibration/src/blm-partial-results.geo.entity.ts

```diff
@@ -0,0 +1,45 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+const tableName = `blm_partial_results`;
+
+@Entity(tableName)
+export class BlmPartialResultEntity {
```

Comment:
This entity is implementation details to calibration runner and do not have to be exposed in `lib`